include/gsl/span

```diff
@@ -262,22 +289,22 @@ namespace details
         // algorithm calls
         friend constexpr void _Verify_range(span_iterator lhs, span_iterator rhs) noexcept
         { // test that [lhs, rhs) forms a valid range inside an STL algorithm
-            Expects(lhs.span_ == rhs.span_        // range spans have to match
-                    && lhs.index_ <= rhs.index_); // range must not be transposed
+            Expects(lhs.begin_ == rhs.begin_    // range spans have to match
+                    && lhs.end_ <= rhs.end_); // range must not be transposed
```

Comment:
This has been incorrectly modified. It should be validating that begin and end are the same, and then the indices should be `<=`. That is, what used to be two tests needs to be three (because of the removal of the parent pointer).